compiler/rustc_middle/src/mir/interpret/value.rs

```diff
@@ -169,19 +169,16 @@ impl<'tcx, Tag> Scalar<Tag> {
     pub fn erase_tag(self) -> Scalar {
         match self {
             Scalar::Ptr(ptr) => Scalar::Ptr(ptr.erase_tag()),
-            Scalar::Raw(int) => Scalar::Raw(int),
+            Scalar::Int(int) => Scalar::Int(int),
         }
     }
 
     #[inline]
     pub fn null_ptr(cx: &impl HasDataLayout) -> Self {
-        Scalar::Raw(ScalarInt::null(cx.data_layout().pointer_size))
+        Scalar::Int(ScalarInt::null(cx.data_layout().pointer_size))
     }
 
-    #[inline]
-    pub fn zst() -> Self {
-        Scalar::Raw(ScalarInt::zst())
-    }
+    pub const ZST: Self = Scalar::Int(ScalarInt::ZST);
```

Comment:
Could you move the `const` up? At least that seems to be the usual style, to have them at the top of the impl block.

Reply:
Done